packages/components/button/stories/ButtonGroup.stories.tsx

```diff
@@ -33,6 +33,7 @@ export default {
 export const basic: Story<ButtonGroupProps> = (args) => {
   return (
     <ButtonGroup {...args}>
+      {false && <Button variant="secondary">Button</Button>}
```

Comment:
Do you want to leave it, or just forgot to remove?

Reply:
to serve as example, and also if kind of test that it's working 😄